server/README.md

```diff
@@ -61,9 +61,13 @@ The server executable can be built with `nix build .#cardano-trasaction-lib-serv
     - `application/json`
 
 - Bytes to hash as hexadecimal string (`application/json;charset=utf-8`, `application/json`):
+- Hashing method from PlutusTx to hash with; one of `sha2_256`, `sha3_256`, `blake2b_256`
```

Comment:
Did you write this manually? We generate the docs using `servant-docs`, so it will get lost the next time we update it

Reply:
makes sense, I didn't know that, yeah, I wrote those manually, didn't know